presto-native-execution/CMakeLists.txt

```diff
@@ -14,7 +14,8 @@ cmake_minimum_required(VERSION 3.10)
 # set the project name
 project(PrestoCpp)
 
-set(VELOX_ROOT ${CMAKE_BINARY_DIR}/velox)
+# NB: uses CMAKE_SOURCE_DIR to support out-of-source builds
+set(VELOX_ROOT ${CMAKE_SOURCE_DIR}/velox)
```

Comment:
Unrelated change: wanted to support out-of-source builds when testing. Will remove before submitting.